src/SFML/Graphics/Shader.cpp

```diff
@@ -737,7 +737,7 @@ void Shader::bind(const Shader* shader)
     else
     {
         // Bind no shader
-        glCheck(GLEXT_glUseProgramObject(0));
+        glCheck(GLEXT_glUseProgramObject({}));
```

Comment:
Would `nullptr` make sense here?

Reply:
Not sure why `clang-tidy` is looking for `nullptr` here when the argument is a `GLuint`

Reply:
```
/Users/thrasher/Projects/sfml/src/SFML/Graphics/Shader.cpp:740:42: error: cannot initialize a parameter of type 'GLhandleARB' (aka 'void *') with an rvalue of type 'int'
        glCheck(GLEXT_glUseProgramObject(1));
                                         ^
```
On my machine it's expecting a `void*`.